src/map.rs

```diff
@@ -1525,15 +1528,22 @@ where
 }
 
 #[inline]
+#[cfg(feature = "std")]
 /// Returns the number of physical CPUs in the machine (_O(1)_).
```

Comment:
Why does this say `(_O(1)_)`?

Reply:
My idea was to indicate that this takes constant time, and the only reason why this is not a `cons` is because `NCPU` need to initialized.

Reply:
Ah, this came across as "the number of physical CPUs is O(1)", which doesn't make much sense. I would probably just delete that part of the comment :)

Reply:
No problem.